pkg/systemlogmonitor/logwatchers/types/log_watcher.go

```diff
@@ -36,6 +36,8 @@ type WatcherConfig struct {
 	// PluginConfig is a key/value configuration of a plugin. Valid configurations
 	// are defined in different log watcher plugin.
 	PluginConfig map[string]string `json:"pluginConfig,omitempty"`
+	// Skip the black list by simple filter to avoid running more efforts in regex
+	BlackList []string `json:"blackList,omitempty"`
```

Comment:
```suggestion
	// Skip the log lines containing any of the strings in the list to avoid running unnecessary regex.
	SkipList []string `json:"skipList,omitempty"`
```

Reply:
done